docs/decisions/0003-reduce-bokchoy-testing.rst

```diff
@@ -0,0 +1,58 @@
+Status
+~~~~~~
+Draft
+
+
+Context
+~~~~~~~
+
+edx-platform bokchoy tests are slow, flaky and difficult to debug.  A quick assessment of their value shows that they might be more trouble than they are worth.  And that we might get the same benefit with far fewer tests.
+
+Baseline Data:(Last 7 days)
+---------------------------
+
+    Total number of builds: 253(across 106 PRs)
+
+    Failures: 49(across 24 PRs)
+        True Failures: 10(across 6 PRs)
+        Failures that wouldn’t be caught by other test: 3(on 1 PR)
+
+Color
+=====
+
+    Of the real failures found, there was one PR which had a failure that was only found via bokchoy and a115 tests.
+        - This PR made a JS change which would have broken many pages from loading.
+
+Recommendation
+--------------
+
+    As an experiment, we should not run bokchoy tests but continue to run a11y tests which will reduce the total number of tests significantly but continue to act as a smoke test for issues that can be caused by the fact that our frontend in edx-platform is still quite highly coupled together.
```

Comment:
What do you think of this update?
```
As an experiment, we will only run a suite of smoke tests in bokchoy that ensure the frontend is not entirely broken.

For the experiment, we will use the a11y bokchoy tests as simple stand-in for a suite of smoke tests, because it is already a much smaller suite of happy path tests.

During the experiment, if we find we are missing coverage via a regression, we will first add a missing Python or JavaScript unit test where possible.  Only if this isn't possible would we add to the smoke suite of bokchoy tests.
```